mutation-testing/filter-pr/action.yml

```diff
@@ -0,0 +1,98 @@
+name: Filter PR Mutants
+
+runs:
+  using: 'composite'
+
+  steps:
+    # Cleanup Runner
+    - name: Cleanup Runner action repo
+      id: runner_cleanup
+      uses: stacks-network/actions/cleanup/disk@main
+
+    # Checkout the stacks-core code
+    - name: Checkout the stacks core repo
+      id: git_checkout_stacks_core
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Relative diff
+      shell: bash
+      run: |
+        git branch -av
+        git diff origin/${{ github.base_ref }}.. | tee git.diff
+
+    # Checkout the actions code
+    - name: Get the action repo code
+      id: git_checkout_actions
+      uses: actions/checkout@v3
+      with:
+        repository: stacks-network/actions
+        ref: main
+        path: ./actions-repo
+
+    - name: Copy the git diff file to scripts folder
+      shell: bash
+      run: |
+        mv git.diff ./actions-repo/mutation-testing/shell-scripts/
+
+    - uses: Swatinem/rust-cache@v2
+
+    - name: Install cargo mutants crate
+      shell: bash
+      run: cargo install cargo-mutants
+
+    - name: Remove deleted file lines from git.diff file
+      shell: bash
+      run: ./remove-deleted-file-lines.sh
```

Comment:
Is this necessary? Shouldn't `cargo mutants` be able to read a diff and know test deleted functions?